roles/zabbix_proxy/tasks/Debian.yml

```diff
@@ -1,103 +1,7 @@
 ---
-- name: "Debian | Set short version name"
-  ansible.builtin.set_fact:
-    zabbix_short_version: "{{ zabbix_proxy_version | regex_replace('\\.', '') }}"
-    zabbix_underscore_version: "{{ zabbix_proxy_version | regex_replace('\\.', '_') }}"
-  tags:
-    - always
-
 - name: "Debian | Set some facts for Zabbix"
   ansible.builtin.set_fact:
     datafiles_path: /usr/share/doc/zabbix-sql-scripts/{{ zabbix_proxy_db_long }}
   tags:
     - install
     - config
```

Comment:
You can kill off this code aswell, it's zabbix-5 related and goes completely unused.

Reply:
done